.github/CODEOWNERS

```diff
@@ -51,3 +51,5 @@ Python/bootstrap_hash.c       @python/crypto-team
 **/*decimal*                  @rhettinger @skrah
 
 **/*idlelib*                  @terryjreedy
+
+**/*typing*                   @gvanrossum @ilevkivslyi
```

Comment:
Thanks for making the PR! There is a small typo: it should be ``@ilevkivskyi`` (the third letter from the end should be ``k``, not ``l``).

Reply:
Dang. And I was trying to be so careful! :-) Fixed now.